utils/management_commands.py

```diff
@@ -0,0 +1,59 @@
+#
+# Freesound is (c) MUSIC TECHNOLOGY GROUP, UNIVERSITAT POMPEU FABRA
+#
+# Freesound is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Freesound is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     See AUTHORS file.
+#
+
+import json
+import logging
+import os
+import sys
+import time
+
+from django.core.management.base import BaseCommand
+
+
+commands_logger = logging.getLogger('commands')
+
+
+class LoggingBaseCommand(BaseCommand):
```

Comment:
would be nice to have some documentation to know what this command does, when it should be used, and how to use it

Reply:
Added ebb16b24e48d3e8da429a01a56a0cac3e8ad9e71